v3.0/how-to/maintain/tidb-binlog.md

```diff
@@ -100,7 +100,7 @@ Command example:
     bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd update-pump -node-id ip-127-0-0-1:8250 -state paused
     ```
 
-    This command modifies the state of Pump or Drainer saved in the Placement Driver (PD).
+    This command modifies the state of Pump or Drainer saved in the Placement Driver (PD) and is only used if Pump or Drainer service fails to function properly. 
```

Comment:
From the context, the reader must use this command very carefully. This information would be better understood if it is in the note format and active voice is used.
```suggestion
    > **Note:**
    >
    > This command modifies the state of Pump or Drainer saved in the Placement Driver (PD).
    > Use this command **only** when the Pump or Drainer service fails to function properly. 
```